packages/@vue/cli/lib/util/clearConsole.js

```diff
@@ -16,12 +16,18 @@ exports.generateTitle = async function (checkUpdate) {
   }
   if (checkUpdate && semver.gt(latest, current)) {
     if (process.env.VUE_CLI_API_MODE) {
-      title += chalk.green(` 🌟️ Update available: ${latest}`)
+      title += chalk.green(` 🌟️ New version available: ${latest}`)
     } else {
-      title += chalk.green(`
-┌────────────────────${`─`.repeat(latest.length)}──┐
-│  Update available: ${latest}  │
-└────────────────────${`─`.repeat(latest.length)}──┘`)
+      const upgradeMessage = `New version available ${chalk.magenta(current)} → ${chalk.green(latest)}\n` +
+        `Run ${chalk.yellow(`npm i -g ${require('../../package.json').name}`)} to update!`
```

Comment:
It should take into account if it's installed using `npm`, `yarn` or `pnpm`, otherwise it will actually cause trouble.

Reply:
Good point. How do we differentiate the two cases though (I don't think we should care for `pnpm`, as it's not even listed on the official docs).

Reply:
Hmm, actually I think something like "…or yarn global add @vue/cli` will do just fine.

Reply:
> as it's not even listed on the official docs

It's officially supported though.

> Hmm, actually I think something like "…or yarn global add @vue/cli` will do just fine.

I think having a tiny help message that doesn't really help (things might go wrong if the user picks an incorrect "choice") is bad. We may display a link to a section in the docs explaining how to upgrade vue-cli though (I think that's the way `now` is doing it).

Reply:
Do we have such a section?

Reply:
@phanan You can now use this. https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli/lib/util/ProjectPackageManager.js#L66. Let's get this merged once you fix this issue.

Reply:
@pksunkara The result may be incorrect. Better to test `__dirname` against `npm config get prefix` / `yarn global dir`